pkg/wakaama/Makefile.include

```diff
@@ -27,5 +19,40 @@ ifneq (,$(or $(CONFIG_LWM2M_WITH_LOGS),$(filter -DCONFIG_LWM2M_WITH_LOGS=1,$(CFL
   CFLAGS += -DLWM2M_WITH_LOGS=1
 endif
 
+# NOTE: "Default CoAP block size; Used if not set on a per-target basis"
+ifeq (,$(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE))
+  CFLAGS += -DLWM2M_COAP_DEFAULT_BLOCK_SIZE=1024
+else
+
+  ifeq ($(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE), 4)
+      COAP_BLOCK_SIZE := 16
+  endif
+
+  ifeq ($(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE), 5)
+      COAP_BLOCK_SIZE := 32
+  endif
+
+  ifeq ($(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE), 6)
+      COAP_BLOCK_SIZE := 64
+  endif
+
+  ifeq ($(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE), 7)
+      COAP_BLOCK_SIZE := 128
+  endif
+
+  ifeq ($(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE), 8)
+      COAP_BLOCK_SIZE := 256
+  endif
+
+  ifeq ($(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE), 9)
+      COAP_BLOCK_SIZE := 512
+  endif
+
+  ifeq ($(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE), 10)
+      COAP_BLOCK_SIZE := 1024
+  endif
+  CFLAGS += -DLWM2M_COAP_DEFAULT_BLOCK_SIZE=$(COAP_BLOCK_SIZE)
```

Comment:
Can't we set it as `-DLWM2M_COAP_DEFAULT_BLOCK_SIZE=(1<<$(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE))` if `CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE` is defined?

Reply:
The shift would be done by the preprocessor, but double-check whether it works well with the -D injection

Reply:
This seems to at least compile without issues:
```makefile
# NOTE: "Default CoAP block size; Used if not set on a per-target basis"
ifeq (,$(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE))
  CFLAGS += -DLWM2M_COAP_DEFAULT_BLOCK_SIZE=1024
else
  CFLAGS += -DLWM2M_COAP_DEFAULT_BLOCK_SIZE='1<<$(CONFIG_LWM2M_COAP_DEFAULT_BLOCK_SIZE)'
endif
```

Reply:
tested it, it sets the correct value i changed it.